### PR TITLE
The Ark of the Clockwork Justicar can now be constructed even if Ratvar's rise is not an objective

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: The Ark of the Clockwork Justicar can now be constructed even if Ratvar's rise is not the objective. 
experiment: Accordingly, its behavior is different; instead of summoning Ratvar, it will convert a massive amount of the station, and should effectively instantly complete the clockwork cult's objectives.
/:cl:

Multiple Ratvars no longer break checks for Ratvar being summoned if one is killed.
I came up with this in the middle of the night seven hours ago from when this pr was opened and coded it in about an hour and a half.

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_ratvar.dm [hunk 1]
```diff
@@ -120,31 +124,59 @@
...
+					var/message = text2ratvar("Behold")
+					world << "<span class='ratvar'>\"[message]!\"</span>"
+					world << "<span class='inathneq_large'>\"[message]!\"</span>"
+					world << "<span class='sevtug_large'>\"[message]!\"</span>"
+					world << "<span class='nezbere_large'>\"[message]!\"</span>"
+					world << "<span class='nzcrentr_large'>\"[message]!\"</span>"
+					world << 'sound/magic/clockwork/invoke_general.ogg'
+					var/x0 = startpoint.x
+					var/y0 = startpoint.y
+					for(var/I in spiral_range_turfs(255, startpoint))
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: This loop isn't particularly cheap, but it is a lot cheaper than an equally-sized explosion, and it's not 'not cheap' for the reason you might assume looking at it:
It makes a lot of pooled overlays, which get deleted via a timer, and mostly slows the timer subsystem down.

File: code/game/gamemodes/clock_cult/clock_ratvar.dm [hunk 2]
```diff
@@ -120,31 +124,56 @@
...
+				if(ratvar_portal)
+					new/obj/structure/clockwork/massive/ratvar(startpoint)
+				else
+					world << "<span class='ratvar'>\"[text2ratvar("Behold")]!\"</span>\n<span class='inathneq_large'>\"[text2ratvar("See Engine's mercy")]!\"</span>\n\
+					<span class='sevtug_large'>\"[text2ratvar("Observe Engine's design skills")]!\"</span>\n<span class='nezbere_large'>\"[text2ratvar("Behold Engine's light")]!!\"</span>\n\
+					<span class='nzcrentr_large'>\"[text2ratvar("Gaze upon Engine's power")]!\"</span>"
+					world << 'sound/magic/clockwork/invoke_general.ogg'
+					var/x0 = startpoint.x
+					var/y0 = startpoint.y
+					for(var/I in spiral_range_turfs(255, startpoint))
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: This loop isn't particularly cheap, but it is a lot cheaper than an equally-sized explosion, and it's not 'not cheap' for the reason you might assume looking at it:
It makes a lot of pooled overlays, which get deleted via a timer, and mostly slows the timer subsystem down.

**Comment by Kromgar** [comment]
most of the time when the arc gets placed most of the crew is dead or converted anyway i dont see the point

**Comment by ChangelingRain** [comment]
tl;dr converting 33% of the crew is harder than summoning ratvar

**Comment by evsey9** [comment]
Noooooo
Make it spawn ratvar
I still want ratvar
